src/core/Collector.js

```diff
@@ -1,30 +1,20 @@
+/* eslint no-await-in-loop: 0 */
 export default class Collector {
   constructor(transport, collectors) {
     this.transport = transport;
     this.collectors = Array.isArray(collectors) ? [].concat(collectors) : [];
   }
 
-  add(collectorRule) {
-    this.collectors.push(collectorRule);
-
-    return this;
-  }
-
-  clone() {
-    return new Collector(this.transport, this.collectors);
-  }
-
   async collect() {
     let event = {};
 
-    this.collectors.map(async collectorRule => {
-
-      event = await collectorRule.prepare(event);
+    for (let i = 0; i < this.collectors.length; i++) {
```

Comment:
@robertdumitrescu `i++` is causing a warning in ESLint. update it with `i+=1` or put a comment on top of the file to prevent this warning

Reply:
Fixed